Cubical/Foundations/Logic.agda

```diff
@@ -35,6 +35,14 @@ inr x = ∣ D.inr x ∣
                                ; (D.inr x) → g x
                                })
 
+-- Mere equality
+
+infix 30 _≡m_
+
+_≡m_ : {A : Set} (x y : A) → Ω
```

Comment:
Hmm, isn't Ω just hProp? I would not have two names for it... Also I don't like to call it Ω, it's not really a subobject classifier (because of predicativity) right?

Reply:
it's hProp at zero, yeah.

We should make most of this polymorphic, but not in this PR. When we do that we can remove Ω?

Reply:
Sounds good, I'll make an issue for it